db/migrate/20181228115540_add_default_location.rb

```diff
@@ -0,0 +1,11 @@
+class AddDefaultLocation < ActiveRecord::Migration
+  def up
+    country = Spree::Country.find_by_iso(ENV['DEFAULT_COUNTRY_CODE'])
+    state = country.states.first
+    Spree::StockLocation.create!(name: 'OFN default', country_id: country.id, state_id: state.id)
```

Comment:
Why did you go with `OFN default` instead of `default` as specified in the wiki? I don't really mind. I think we came up with `default`, because it's used by Spree sample data:
https://github.com/openfoodfoundation/spree/blob/f55722b38db7e706a8521c9091a0e00119bb4d20/sample/db/samples/stock.rb#L3

Reply:
I didn't know we stated its name in the wiki. I just wanted to differentiate from Spree's stuff.

How does that Spree code relate to OFN? AFAIK that's Spree's sample data and there is no default location as per its migrations.

Reply:
for the record, openfoodfoundation/spree:sample/db/samples/stock.rb@f55722b#L3 is not executed with Spree's seed data (the one that we run when executing `bundle exec rake db:reset`) but from Spree's CLI, which I think is used to set up new Spree projects.